Astronomy/Astronomy/Views/PhotoCollectionViewCell.m

```diff
@@ -14,14 +14,24 @@ @implementation PhotoCollectionViewCell
 
 -(void)layoutSubviews {
     [super layoutSubviews];
+
+    _imageBackgroundView.layer.cornerRadius = 8;
+    _imageBackgroundView.layer.shadowColor = [UIColor lightGrayColor].CGColor;
+    _imageBackgroundView.layer.shadowOpacity = 1;
+    _imageBackgroundView.layer.shadowOffset = CGSizeMake(0, 0);
+    _imageBackgroundView.layer.shadowRadius = 3;
+    _imageBackgroundView.layer.masksToBounds = NO;
+    _imageBackgroundView.backgroundColor = [UIColor whiteColor];
+    _imageView.layer.cornerRadius = 8;
+    _imageView.clipsToBounds = YES;
```

Comment:
Generally when we're not in `init`, we should use `self.propertyName` rather than `_propertyName`, or we might get some unintended effects (or more likely miss out on some intended effects).

Reply:
I also got a crash on Sol 10, I clicked on the switch camera.

```objc
2020-05-21 10:18:46.531233-0400 Astronomy[87747:15387278] *** Terminating app due to uncaught exception 'NSRangeException', reason: '*** -[__NSArrayM objectAtIndexedSubscript:]: index 71 beyond bounds [0 .. 25]'
```

<img width="400" alt="Screen Shot 2020-05-21 at 10 19 02 AM" src="https://user-images.githubusercontent.com/371902/82569117-ca79fa00-9b4d-11ea-932d-1ace3ddec427.png">